pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -808,7 +848,49 @@ log(404, "NOT_FOUND", "this", "page", "is", "not", "found");
 ```
 
 ## `[@bs.inline]`
-TODO
+
+Especially for library authors, `bs.inline` could be interesting. It tells the compiler explicitly to inline a constant directly. 
```

Comment:
Again, just get to the point, explain details later if necessary:

~~~suggestion
A binding marked with `[@bs.inline]` will tell the compiler to copy (inline) its value in every place the binding is being used. This is especially useful for cases where e.g. string constants can't be used as variables, but need to be present as constant values.

For example,...
~~~